### PR TITLE
WIP(builder):couple builder types  and add dockerbuilder pod yaml

### PR DESCRIPTION
**Pull request by smothiki**
SIngle server that handles both requests for dockerfile apps and buildpack apps.
identifies type if app and spins up necessary POD to build the artifact

### CODE REVIEW

**Comment by smothiki** [comment]
Needs integration with dockerbuilder . Tested for buildpack apps. Kept fetcher component for now.
cc @arschles

File: manifests/deis-builder-rc.yaml [hunk 1]
```diff
@@ -18,9 +18,20 @@ spec:
         app: deis-builder
     spec:
       containers:
+<<<<<<< HEAD:manifests/deis-bpbuilder-rc.yaml
```
Comment (arschles) [inline manifests/deis-builder-rc.yaml]: can you fix these conflicts

**Comment by smothiki** [comment]
@arschles @technosophos  PTAL. This is working for buildpack types. If we can get this merged @arschles can work on S3 endpoint and dockerbuilder integration

**Comment by arschles** [comment]
@smothiki LGTM, but haven't tested slugbuilder. I currently have https://github.com/smothiki/builder/pull/2 open on this branch and am testing. FYI if you merge this before I finish, I'll rebase against `deis:master` and open a PR over there

**Comment by technosophos** [comment]
My favorite kind of commit! Lots of red!

Yeah, this looks much cleaner. Are you ready to merge? I'll LGTM if you are.

**Comment by smothiki** [comment]
Okay LGTM's please

**Comment by technosophos** [comment]
This looks great, @smothiki

**Pull request merged**